app/models.py

```diff
@@ -417,3 +417,53 @@ class MatomoDailyGetSiteSearchKeywords(db.Model):
 
     def __repr__(self):
         return '<MatomoDailyGetSiteSearchKeywords {}>'.format(self.id)
+
+
+class GithubDailyClonesCount(db.Model):
+    """
+    Provides GitHub daily clones' count to be stored in database table
+    github_daily_clones_count.
+
+    repo         = name of the GitHub repository
+    date         = date associated to the count
+    timestamp    = timestamp associated to the count
+    count        = number of clones of the GitHub repository for a given date
+    unique_count = number of unique clones for a given date
+    """
+
+    __tablename__ = "github_daily_clones_count"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    repo = db.Column(db.String(256))
+    date = db.Column(db.String(12))
```

Comment:
Why not make the `date` and `timestamp` columns of the new models with types `db.Date` and `db.DateTime`? Probably doesn't make a huge difference and would take some conversion in `_get_repo_analytics`, though, so up to you.

Reply:
I totally agree with you on that one. Unfortunately, I tried that at the time I designed the database tables for the Matomo analytics and there were issues configuring the date field as a date. I don't remember the specific issues I had but it was impossible to get the date being of type date. I think it was with SQLite that it failed but I don't remember very well. I just remember it did not work, haha.

Reply:
Fair enough, that's a good reason!